lib/ui/package.json

```diff
@@ -4,7 +4,7 @@
     "ember-addon"
   ],
   "dependencies": {
-    "ember-cli-babel": "^7.1.2",
```

Comment:
Do we even need this `package.json`? If we omit it, would it use the deps from the root `package.json`?

Reply:
We need it for the `ember-addon` stuff but I switched the deps to `*` so it just uses whatever is at the root